docs/src/config.ts

```diff
@@ -50,36 +50,20 @@ export const sidebar = [
       { text: 'Renderer Reference', link: 'reference/renderer-reference' },
     ],
   },
-  // To add once rest of the site is complete
-  // see https://github.com/snowpackjs/astro-docs/issues/9
-  // {
-  //   text: 'Integrations',
-  //   link: 'integrations',
-  //   children: [
-  //     { text: 'Deploy Astro', link: 'integrations/deploy-astro' },
-  //     { text: 'Data Sources / CMS', link: 'integrations/data-sources-cms' },
-  //     { text: 'State Management', link: 'integrations/state-management' },
-  //     {
-  //       text: 'Styles & CSS Libraries',
-  //       link: 'integrations/styles-and-css-libraries',
-  //     },
-  //     { text: 'Developer Tools', link: 'integrations/developer-tools' },
-  //   ],
-  // },
 ];
 
-export const site = {
+export const SITE = {
   title: 'Astro Documentation',
   description: 'Build faster websites with less client-side Javascript.',
-  ogLocale: 'en_US',
+}
```

Comment:
to provide context for why this was named ogLocale and not "locale" is because it's a different locale formatting than what you would use in the lang="" attribute on html: "en-us" vs "en_US" - I think your refactoring works great though!

Reply:
Yea good call. the intention is that it's `OPEN_GRAPH.locale` which hopefully makes that clear as well.